### ISSUE TITLE
Layout of results is ambiguous

### ISSUE THREAD

**Issue by NadiaSteyn** [TrystanScottLambert/Cross_Matching#5]
I'm cross-matching a list of 17 sources. This is what the result says:

----------------------------------
Cross matching done: 

13 sources with one match
2 sources with multiple possible matches
2 sources with no matches
----------------------------------

**15** of my 17 sources found a match, but it is not clear whether it states "13 matches of which 2 have multiple matches", or "13 matches plus 2 with multiple matches". The results should rather start with **total** matches, then a break-down of how many have 1 match / multiple matches. For example:

----------------------------------
Cross matching done: 

15 matches
    - 13 sources with one match
    - 2 sources with multiple possible matches
2 sources with no matches
----------------------------------

**Comment by TrystanScottLambert** [comment]
Good idea. @NadiaSteyn, can you please implement the above feature? Awaiting the PR with bated breath.
